--- gftp-2.0.16.orig/docs/sample.gftp/bookmarks	2002-08-05 21:46:57.000000000 +0200
+++ gftp-2.0.16/docs/sample.gftp/bookmarks	2003-11-30 21:02:17.000000000 +0100
@@ -172,6 +172,26 @@
 password=@EMAIL@
 account=
 
+[PLD Sites/PLD Ac]
+hostname=ftp.pld-linux.org
+port=21
+protocol=FTP
+remote directory=/dists/ac
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[PLD Sites/PLD Nest-test]
+hostname=ftp.nest.pld-linux.org
+port=21
+protocol=FTP
+remote directory=/test
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
 [RedHat Sites/FalseHope RPMs]
 hostname=ftp.falsehope.com
 port=21
