--- gftp-2.0.17/configure.in.orig	2004-03-29 02:00:10.000000000 +0200
+++ gftp-2.0.17/configure.in	2004-10-23 20:03:32.147303184 +0200
@@ -41,7 +41,7 @@
 AM_C_PROTOTYPES
 AC_C_CONST
 AC_C_INLINE
-AC_INTL_PRINTF
+dnl AC_INTL_PRINTF
 
 AM_MAINTAINER_MODE
 
@@ -55,7 +55,7 @@
 AC_HEADER_SYS_WAIT
 AC_CHECK_HEADERS(fcntl.h libutil.h limits.h malloc.h pty.h strings.h sys/ioctl.h sys/time.h unistd.h stdint.h sys/mkdev.h)
 
-AC_TYPE_SOCKLEN_T
+dnl AC_TYPE_SOCKLEN_T
 AC_TYPE_MODE_T
 AC_TYPE_PID_T
 AC_TYPE_SIZE_T
