diff -aurN gftp-2.0.15rc1.orig/docs/sample.gftp/bookmarks gftp-2.0.15rc1/docs/sample.gftp/bookmarks
--- gftp-2.0.15rc1.orig/docs/sample.gftp/bookmarks	2002-08-05 21:46:57.000000000 +0200
+++ gftp-2.0.15rc1/docs/sample.gftp/bookmarks	2003-07-25 14:30:28.000000000 +0200
@@ -172,6 +172,26 @@
 password=@EMAIL@
 account=
 
+[PLD Sites/PLD Ra]
+hostname=ftp.pld-linux.org
+port=21
+protocol=FTP
+remote directory=/dists/Ra
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[PLD Sites/PLD Nest-test]
+hostname=ftp.nest.pld-linux.org
+port=21
+protocol=FTP
+remote directory=/test
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
 [RedHat Sites/FalseHope RPMs]
 hostname=ftp.falsehope.com
 port=21
