--- gftp-2.0.18/docs/gftp.desktop_old	2004-11-03 23:11:57.000000000 +0100
+++ gftp-2.0.18/docs/gftp.desktop	2004-10-23 00:00:00.000000000 +0200
@@ -1,12 +1,32 @@
 [Desktop Entry]
-Version=1.0
 Encoding=UTF-8
 Name=gFTP
-GenericName=gFTP
-Comment=Download and upload files using multiple file transfer protocols
-Exec=gftp %u
-TryExec=gftp
+Name[br]=gFTP
+Name[da]=gFTP
+Name[de]=gFTP
+Name[es]=gFTP
+Name[fr]=gFTP
+Name[ja]=gFTP
+Name[ko]=gFTP
+Name[nl]=gFTP
+Name[pl]=gFTP
+Name[pt_BR]=gFTP
+Name[zh_TW]=gFTP
+Comment=Multithreaded ftp client
+Comment[br]=Cliente de ftp multithreaded
+Comment[da]=Flertrådet ftpklient
+Comment[de]=Multithreaded FTP Programm
+Comment[es]=Cliente FTP multihilo
+Comment[fr]=Client ftp multithread 
+Comment[ja]=マルチスレッドFTPクライアント
+Comment[ko]=다중쓰레드 FTP 프로그램
+Comment[nl]=Multithreaded ftp programma
+Comment[pl]=Wielowątkowy klient FTP
+Comment[pt_BR]=Cliente FTP multi-tarefa
+Comment[zh_TW]=多線緒  ftp client
+Exec=gftp
 Icon=gftp.png
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Application;GTK;GNOME;Network;FileTransfer;
+Categories=GTK;Network;FileTransfer;
+# vi: encoding=utf-8
