diff -aurN gftp-2.0.17.orig/configure gftp-2.0.17/configure
--- gftp-2.0.17.orig/configure	2004-03-29 02:00:22.000000000 +0200
+++ gftp-2.0.17/configure	2004-10-23 20:37:28.320757768 +0200
@@ -973,7 +973,7 @@
 
 
 
-ALL_LINGUAS="am az ar be bg bn ca cs da de el es fi fr ga hr hu it ja ko mk ml ms nl no pl pt pt_BR ro ru sk sr sr@Latn sv ta th tr uk zh_CN zh_TW"
+ALL_LINGUAS="am az ar be bg bn ca cs da de el es fi fr ga hr hu it ja ko mk ml ms nb nl pl pt pt_BR ro ru sk sr sr@Latn sv ta th tr uk zh_CN zh_TW"
 
 
 # Make sure we can run config.sub.
diff -aurN gftp-2.0.17.orig/configure.in gftp-2.0.17/configure.in
--- gftp-2.0.17.orig/configure.in	2004-03-29 02:00:10.000000000 +0200
+++ gftp-2.0.17/configure.in	2004-10-23 20:37:07.099983816 +0200
@@ -28,7 +28,7 @@
 AC_SUBST(VERSION)
 AC_SUBST(PREFIX)
 
-ALL_LINGUAS="am az ar be bg bn ca cs da de el es fi fr ga hr hu it ja ko mk ml ms nl no pl pt pt_BR ro ru sk sr sr@Latn sv ta th tr uk zh_CN zh_TW"
+ALL_LINGUAS="am az ar be bg bn ca cs da de el es fi fr ga hr hu it ja ko mk ml ms nb nl pl pt pt_BR ro ru sk sr sr@Latn sv ta th tr uk zh_CN zh_TW"
 
 AC_CANONICAL_HOST
 AC_PROG_CC
